make build fails

```
$ make build
sed "s/GO_VERSION/1.9/g" < build/build-image/Dockerfile | \
  docker build -t scbuildimage -f - .
unable to prepare context: unable to evaluate symlinks in Dockerfile path: lstat /home/kargakis/kube/src/github.com/kubernetes-incubator/service-catalog/-: no such file or directory
make: *** [.scBuildImage] Error 1
```

On Centos 7

--- a/docs/devguide.md
+++ b/docs/devguide.md
@@ -65,7 +65,7 @@ document.
 
 At a minimum you will need:
 
-* [Docker](https://www.docker.com) installed locally
+* [Docker](https://www.docker.com) 17.05+ installed locally
 * GNU Make
 * [git](https://git-scm.com)
 

